README.md

```diff
@@ -179,10 +179,100 @@ If you're calling a **GET** on `/entities` , you're asking for a list of entitie
 - **GET**  `/media/:media_id/update_comment?text=comment%20new%20text`
 > Deleting a comment on a media item:
 
-- **GET** `/media/:media_id/delete_comment?id=comment1`
+- **GET** `/media/:media_id/delete_comment?id=:id`
 
 
 ##### Reasoning
-Using the right HTTP Verb on each resource will make clearer what the resource is made for and what we should expect it to do, receive and return. 
+Using the right HTTP Verb on each endpoint will make clearer what the endpoint is made for and what we should expect it to do, receive and return. 
+
+---
+
+#### Use Nouns for endpoints names (URLs)
+> Your endpoints names should be always nouns, not verbs.
+
+##### Examples
+
+###### Good
+
+> Getting a user
+
+- **GET** `/users/:id`
+  The endpoint retrieves the user with the id you pass along in the usl.
+
+> Creating a user:
+
+- **POST** `/users`
+  This creates a user with the info you pass along in the request body.
+
+> Deleting a post
+
+- **DELETE** `/users/:id`
+  This Deletes the user with the id you pass along in the url
+
+###### Bad
+
+> Getting a post
+
+- **GET** `/get/posts?id=:id`
+  This is wrong because you already established that it is a GET. Using /get on the endpoint name is redundant
+
+> Creating a post
+
+- **POST** `/create/post`
+  Same as above, you already estabished that you're going to create something with the **POST** so, again, the create is redundant
+
+> Deleting a post
+
+- **POST** `/delete_post/:id`
+  In thi case not only you're being redundant by putting the verb delete on the endpoint name, but you're doing a wrong redundancy because it is actually a **POST** request and they're not for deleting objects
```

Comment:
thi -> this.
The description is wrong. There is no redundancy here. The problem should be stated as follows:

> In this case you're using **POST** (a verb that's intended to be used for creation) to delete an object. That's semantically wrong. You should use **DELETE**  for that, instead. If you do so, you'll notice that you no longer need to add `delete` to your url and that then your url will clearly identify the object being deleted.